plutus-script-utils/src/Plutus/V1/Utils/Generators.hs

```diff
@@ -0,0 +1,39 @@
+{-# LANGUAGE DataKinds       #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeFamilies    #-}
+
+module Plutus.V1.Utils.Generators
+    ( UnitTest
+    , alwaysSucceedValidator
+    , alwaysSucceedValidatorHash
+    , alwaysSucceedPolicy
+    , someTokenValue
+    ) where
+
+import Plutus.V1.Ledger.Scripts qualified as Ledger
+import Plutus.V1.Ledger.Value (TokenName, Value)
+import Plutus.V1.Ledger.Value qualified as Value
+import PlutusTx qualified
+
+import Plutus.V1.Utils.Contexts qualified as Contexts
+import Plutus.V1.Utils.Typed.Scripts qualified as Scripts
+
+data UnitTest
+instance Scripts.ValidatorTypes UnitTest
+
+alwaysSucceedValidator :: Scripts.TypedValidator UnitTest
+alwaysSucceedValidator = Scripts.mkTypedValidator
+    $$(PlutusTx.compile [|| \_ _ _ -> True ||])
+    $$(PlutusTx.compile [|| wrap ||])
+    where
+        wrap = Scripts.wrapValidator
```

Comment:
This kind of renders `TypedValidator` and `mkTypedValidator` ad absurdum. Why not just have a `Ledger.mkValidatorScript $$(PlutusTx.compile [|| \_ _ _ -> () ||])` here?